nautobot/extras/tests/test_api.py

```diff
@@ -2047,13 +2053,14 @@ def test_run_job_with_valid_task_queue(self, _):
 
     @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
     @mock.patch("nautobot.extras.api.views.get_worker_count", return_value=1)
-    def test_run_job_with_default_queue_with_empty_job_model_task_queues(self, _):
+    def test_run_job_with_default_queue_with_empty_job_model_job_queues(self, _):
         self.add_permissions("extras.run_job")
         data = {
             "task_queue": settings.CELERY_TASK_DEFAULT_QUEUE,
         }
 
         job_model = Job.objects.get_for_class_path("pass.TestPass")
+        job_model.job_queues.set([])
```

Comment:
This looks like the wrong fix to me - shouldn't we instead correct the `data` above to specify a `job_queue`?

Reply:
I think the original test is testing the job without task_queues specified in the class meta and no overrides as well. More specifically, I think it is testing that `valid_queues` will always include the [default](https://github.com/nautobot/nautobot/pull/6154/files/8cd0cdb7575c5bb9bcc53a871b6e00e6210f859d#diff-9a2bdd132c5e3a0e6947134b0ae99c641837ad953eb9f5585633bf33e4a60dc9R602-R603) queue if no queues are specified. And on the `JobInputSerializer`, the field for specifying a queue is still named `task_queue` for backward compatibility reasons.

Reply:
We should enhance `JobInputSerializer` to support a `job_queue` argument as an alternative - possibly a versioned API needed.